packages/common/src/hooks/purchaseContent/useChallengeCooldownSchedule.ts

```diff
@@ -81,36 +51,65 @@ export const formatAudioMatchingChallengesForCooldownSchedule = (
     cooldownChallenges[diff] = {
       ...cooldownChallenges[diff],
       id: c.specifier,
-      label: getAudioMatchingCooldownLabel(c, now),
-      value: (cooldownChallenges[diff]?.value ?? 0) + c.amount
+      value: (cooldownChallenges[diff]?.value ?? 0) + c.amount,
+      ...getCooldownChallengeInfo(c, now)
     }
   })
   return cooldownChallenges
 }
 
-const getAudioMatchingChallengeCooldownSummary = (claimableAmount: number) => ({
-  id: messages.readyToClaim,
-  label: messages.readyToClaim,
-  value: claimableAmount
-})
-
+const TRENDING_CHALLENGE_IDS = new Set(['tt', 'tut', 'tp'])
 /**
- * Custom hook using values specific to $AUDIO matching challenges.
+ * Returns a list of challenges in cooldown period (not claimable yet), and
+ * the total currently claimable amount (for challenges past the cooldown period).
  */
-export const useAudioMatchingChallengeCooldownSchedule = (
+export const useChallengeCooldownSchedule = ({
+  challengeId,
+  multiple
+}: {
   challengeId?: ChallengeRewardID
-) => {
-  const { cooldownChallenges, claimableAmount } =
-    useChallengeCooldownSchedule(challengeId)
+  multiple?: boolean
+}) => {
+  const challenges = useSelector(getUndisbursedUserChallenges)
+    .filter((c) => multiple || c.challenge_id === challengeId)
+    .filter((c) => !TRENDING_CHALLENGE_IDS.has(c.challenge_id))
+    .map((c) => ({ ...c, createdAtDate: dayjs.utc(c.created_at) }))
+
+  // Challenges that are claimable now
+  const claimableChallenges = challenges.filter(isCooldownChallengeClaimable)
```

Comment:
it might be cool to use lodash's "partition" to divide challenges to claimable and cooldown challenges in one function

Reply:
and it might be overkill, but you could also use lodash sum function. so together maybe
```
const [claimableChallenges, cooldownChallenges] = partition(challenges, isChallengeClaimable)
const claimableAmount = sum(claimableChallenges)
const cooldownAmount = sum(cooldownChallenges)
```
or 
```
const cooldownChallengePartition = partition(challenges, isChallengeClaimable)
const [claimableChallenges, cooldownChallenges] = cooldownChallengePartition
const [claimableAmount, cooldownAmount] = cooldownChallengePartition.map(sum)
```

Reply:
very cool